staging/sales_index.json

```diff
@@ -1,4 +1,30 @@
+
 [
+  {
+      "chainId": 3,
+      "saleManagerAddress": "0x000047203100a45635029ec21bbbec5ec53cb6f6",
+      "saleId": "0x81c02c427b98a54aa297b164f192706eeeae051fbdeb8afcd083768701fbd96d",
+      "saleName": "PenJ Softernet Sale",
+      "logo": "https://github.com/penj13/launchlists/blob/patch-1/Bitcoin-Logo.png?raw=true",
+      "favicon": "https://github.com/penj13/launchlists/blob/main/381-3810080_facebook-icon-favicon.png?raw=true",
+      "projectWebsite": "https://tokensoft.io",
+      "saleDescription": "PenJ Softernet Sale is Good",
+      "access": {
+        "limitToRegions": ["AL","DZ","AD","AO","AI","AG","AR","AM","AU","AT","AZ","BS","BH","BD","BB","BY","BE","BZ","BJ","BM","BT","BO","BA","BW","BR","BN","BG","BF","BI","KH","CM","CV","KY","CF","TD","CL","CO","KM","CD","CG","CR","CI","HR","CY","CZ","DK","DJ","DM","DO","EC","EG","SV","GQ","ER","EE","ET","FO","FJ","FI","FR","PF","GA","GM","GE","DE","GH","GI","GR","GD","GT","GG","GN","GW","GY","HT","VA","HN","HK","HU","IS","IN","ID","IQ","IE","IM","IL","IT","JM","JP","JE","JO","KZ","KE","KI","KR","KW","KG","LA","LV","LB","LS","LR","LY","LI","LT","LU","MO","MK","MG","MW","MV","ML","MT","MH","MR","MU","MX","MY","FM","MD","MC","MN","ME","MS","MA","MZ","MM","NA","NR","NP","NL","NZ","NI","NE","NG","NO","OM","PK","PW","PS","PA","PG","PY","PE","PH","PL","PT","PR","QA","RO","RU","RW","KN","LC","MF","VC","WS","SM","ST","SA","SN","RS","SC","SL","SG","SK","SI","SB","SO","ZA","ES","LK","SR","SZ","SE","CH","TW","TJ","TZ","TH","TL","TG","TO","TT","TN","TR","TM","TC","TV","UG","AE","GB","UY","UZ","VU","VE","VN","EH","YE","ZM","ZW","SD","CN","CA","SS","UA"]      },
+      "socials": {
+        "twitter": "https://twitter.com/https://twitter.com/alex_savchik",
+        "discord": "https://discord.gg/tokensoft",
+        "telegram": "https://t.me/follow2themoon"
+      },
+      "documents": [
+        {
+          "name": "Purchase Agreement",
+          "uri": "https://randstorageprod.blob.core.windows.net/sale/SAFT%20Rand%20Presale%202%20Angels%20(TokenSoft).docx-2.pdf",
```

Comment:
this is a weird URL - anything simpler you can use?

Reply:
corrected